tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Preview/Display Marketing Page Preview Capabilities Section.3507.feature

```diff
@@ -5,31 +5,43 @@ Feature: Display Marketing Page Preview Capabilities Section
 
 Background:
 	Given Capabilities exist
-		| CapabilityName          | IsFoundation |
-		| Appointments Management | true         |
-		| Prescribing             | true         |
-		| Workflow                | true         |
-		| Clinical Safety         | false        |
+		| CapabilityName          | IsFoundation | Version | Description | SourceUrl                          |
+		| Appointments Management | true         | 2.0     | AM          | http://appointments.management.com |
+		| Prescribing             | true         | 2.1     | P           | http://prescribing.com             |
+		| Workflow                | true         | 2.2     | W           | http://workflow.com                |
+		| Clinical Safety         | false        | 2.3     | CS          | http://clinical.safety.com         |
 	And Suppliers exist
 		| Id    | SupplierName |
 		| Sup 1 | Supplier 1   |
 	And Solutions exist
-		| SolutionID | SolutionName   | SupplierStatusId | SupplierId |
+		| SolutionId | SolutionName   | SupplierStatusId | SupplierId |
 		| Sln1       | MedicOnline    | 1                | Sup 1      |
 		| Sln2       | TakeTheRedPill | 1                | Sup 1      |
 	And Solutions are linked to Capabilities
-		| Solution    | Capability              |
-		| MedicOnline | Appointments Management |
-		| MedicOnline | Clinical Safety         |
-		| MedicOnline | Workflow                |
+		| Solution    | Capability              | Pass  |
+		| MedicOnline | Appointments Management | True  |
+		| MedicOnline | Clinical Safety         | True  |
+		| MedicOnline | Workflow                | True  |
+		| MedicOnline | Prescribing             | False |
 
 @3507
 Scenario: 1. Sections presented where Capabilities exists
 	When a GET request is made for solution preview Sln1
 	Then a successful response is returned
 	And the response contains the following values
-		| Section      | Field            | Value                                              |
-		| capabilities | capabilities-met | Appointments Management, Clinical Safety, Workflow |
+		| Section      | Field                           | Value                              |
+		| capabilities | capabilities-met[0].name        | Appointments Management            |
```

Comment:
This looks a bit messy with the capabilities met indexer. We may need to revisit this in the future.